### PR TITLE
64653 left nav refresh

### PR DESCRIPTION
**Pull request by aaronransley**
Refreshes visual style of left sidebar navigation, and patches styles + a bit of JS to ensure that they function both on list / index pages, as well as on edit / new / form pages.

### CODE REVIEW

File: app/assets/stylesheets/fae/globals/navigation/_sidenav.scss [hunk 1]
```diff
@@ -1,33 +1,34 @@
 .sidenav {
   @include stack($fae-nav-stack);
   position: absolute;
+  left: 30px;
```
Comment (tshedor) [inline app/assets/stylesheets/fae/globals/navigation/_sidenav.scss]: Same with the `$content-buffer` comment above

File: app/assets/stylesheets/fae/globals/navigation/_sidenav.scss [hunk 2]
```diff
@@ -48,36 +65,77 @@
...
-        @extend .icon-arrow_up:before;
-        color: $c-black;
+        @extend .icon-arrow_down:before;
+        top: 19px;
+        transform: rotate(0);
+        opacity: 1;
+      }
+    }
+  }
+  &.-open.-parent-current {
```
Comment (tshedor) [inline app/assets/stylesheets/fae/globals/navigation/_sidenav.scss]: Merge with previous rule - `&.-parent-current`

File: app/assets/javascripts/fae/navigation/_navigation.js [hunk 3]
```diff
@@ -169,12 +169,13 @@ Fae.navigation = {
...
-      var sidebar_top_offset = (parseInt( $header.outerHeight(), 10) + 20) + 'px';
-      $('#js-sidenav').css('padding-top',  sidebar_top_offset );
+      var sidebar_top_offset = (parseInt( $header.outerHeight(), 10) + 30) + 'px';
+      $('#js-sidenav').css('padding-top', sidebar_top_offset );
 
       $header.sticky({
         placeholder: true,
-        perpetual_placeholder: true
+        perpetual_placeholder: true,
+        ignore_placeholder_offsets: true
```
Comment (tshedor) [inline app/assets/javascripts/fae/navigation/_navigation.js]: This seems to add some unnecessary complexity. Do both have to be true? Couldn't `perpetual_placeholder` just be false? 

If `perpetual_placeholder` works as `false`, then you can remove the added `ignore_placeholder_offsets` option.

File: app/assets/stylesheets/fae/globals/navigation/_base.scss [hunk 4]
```diff
@@ -5,7 +5,7 @@
 .nav {
   top: 0;
   left: 0;
-  width: $sidenav-width;
+  width: $sidenav-width - 30px;
```
Comment (aaronransley) [inline app/assets/stylesheets/fae/globals/navigation/_base.scss]: Have replaced `30px` with `$content-buffer`

File: app/assets/stylesheets/fae/globals/navigation/_sidenav.scss (hunk 2 above)
Comment (aaronransley) [inline app/assets/stylesheets/fae/globals/navigation/_sidenav.scss]: Merged

File: app/assets/stylesheets/fae/globals/navigation/_sidenav.scss (hunk 1 above)
Comment (aaronransley) [inline app/assets/stylesheets/fae/globals/navigation/_sidenav.scss]: 👍

File: app/assets/stylesheets/fae/globals/navigation/_base.scss (hunk 4 above)
Comment (tshedor) [inline app/assets/stylesheets/fae/globals/navigation/_base.scss]: @aaronransley Thanks

**Pull request merged**